include/cantera/thermo/PengRobinson.h

```diff
@@ -173,6 +173,17 @@ class PengRobinson : public MixtureFugacityTP
     void setBinaryCoeffs(const string& species_i, const string& species_j, double a);
     //! @}
 
+    //! Return parameters used by the Peng-Robinson equation of state.
+    /*!
+     * @return
+     *   Returns an AnyMap containing the following key names and values:
+     *   - "aAlpha_mix": The value of m_aAlpha_mix
+     *   - "b_mix": The value of the m_b
+     *   - "daAlpha_dT": The value returned by the daAlpha_dT() method
+     *   - "d2aAlpha_dT2": The value returned by the d2aAlpha_dT2() method
```

Comment:
For better formatting and autolinking by Doxygen, I'd suggest:
```suggestion
     *   - `aAlpha_mix`: The value of #m_aAlpha_mix
     *   - `b_mix`: The value of the #m_b
     *   - `daAlpha_dT`: The value returned by the daAlpha_dT() method
     *   - `d2aAlpha_dT2`: The value returned by the d2aAlpha_dT2() method
```